docs/commands/autoview.md

```diff
@@ -0,0 +1,59 @@
+# autoview
+
+Print the content of the pipeline as a table or list.
+It is the implied or default viewer when none is provided.
+
+When reading a single value, a table or a list, `autoview` will attempt to view it.
+When reading a string that originally comes from a source file it will attempt
+to use `textview`.
+When reading a binary file it will attempt to display its content as hexadecimal
+numbers and the corresponding characters.
```

Comment:
Not 100% sure about "corresponding character" as many hex values drop to a dot `.`.

Reply:
Yeah I think "attempts to" helps here. Yeah this is good